### PR TITLE
Cloud native security overview updates

### LINKED ISSUE #3387: Update Container Workload Protection description on Cloud Native Sec landing page
I received some feedback from Dan on the verbiage describing Cloud workload protection after the PR updating the Cloud Native Security had been merged. Time to incorporate it!

### PR DESCRIPTION
**Pull request by benironside**
Fixes #3387 by incorporating some late-breaking feedback on the cloud native security overview section updates that I made for 8.8.

Preview: [Cloud native security](https://security-docs_3389.docs-preview.app.elstc.co/guide/en/security/master/cloud-native-security-overview.html)

### CODE REVIEW

File: docs/cloud-native-security/cloud-native-security-index.asciidoc [hunk 1]
```diff
@@ -25,13 +25,13 @@ Scans your cloud workloads for known vulnerabilities. When it finds a vulnerabil
...
 
 <<d4c-overview, Read the Container workload protection docs>>.
 
 [discrete]
 == Cloud Workload Protection for VMs
-Helps you monitor and protect your Linux VMs. It captures workload telemetry data containing process, file, and network activity. You can use this data — along with Elastic's out-of-the-box detection rules and machine learning models — to automate threat identification for your cloud VMs. Alerts that are generated from these detections help reduce the time to identify and remediate threats.
+Helps you monitor and protect your Linux VMs. It uses {elastic-defend} to instantly detect and prevent malicious behavior and malware, and captures workload telemetry data for process, file, and network activity. You can use this data — along with Elastic's out-of-the-box detection rules and machine learning models — to automate threat identification for your cloud VMs. Alerts that are generated from these detections help reduce the time to identify and remediate threats.
```
Comment (benironside) [inline docs/cloud-native-security/cloud-native-security-index.asciidoc:34]:
```suggestion
Helps you monitor and protect your Linux VMs. It uses {elastic-defend} to instantly detect and prevent malicious behavior and malware, and captures workload telemetry data for process, file, and network activity. You can use this data with Elastic's out-of-the-box detection rules and machine learning models. Alerts that are generated from these detections help reduce the time to identify and remediate threats.
```

Comment (nastasha-solomon) [reply docs/cloud-native-security/cloud-native-security-index.asciidoc:34]: Just have two more minor suggestions. One is an attribute recommendation and the other is a slight revision to make the last sentence more active:

```suggestion
Helps you monitor and protect your Linux VMs. It uses {elastic-defend} to instantly detect and prevent malicious behavior and malware, and captures workload telemetry data for process, file, and network activity. You can use this data with Elastic's out-of-the-box detection rules and {ml} models. These detections generate alerts that quickly help you identify and remediate threats.
```

**Review by nastasha-solomon — APPROVED** [review]
Just left one suggestion. LGTM overall!

File: docs/cloud-native-security/cloud-native-security-index.asciidoc (hunk 1 above)
Comment (jmikell821) [reply docs/cloud-native-security/cloud-native-security-index.asciidoc:34]: This suggestion ^ is fine.

**Review by jmikell821 — COMMENTED** [review]
Thanks @benironside for chatting about this today. I'm OK w/ @nastasha-solomon's proposed suggestion, then you can merge. Thanks!

**Pull request merged**